recipes/qsymm/run_test.py

```diff
@@ -0,0 +1,9 @@
+import os.path
```

Comment:
Is this file being used?

Reply:
my understanding was that it is run automatically when testing occurs

Reply:
ah, seems I can just run `pytest` and specify this in `meta.yaml`; I thought pip was building the package directly from the source archive, but it seems it's unpacked first? (`pip install .`)

Reply:
Yes, by default the build is ran from a directory containing the extracted tarball, this is then deleted for the testing step. If you want to access the recipe directory you can use `${RECIPE_DIR}`.

If you want to keep files from the the build for the testing step you can [specify `source_files`](https://docs.conda.io/projects/conda-build/en/latest/source/resources/define-metadata.html#source-files).